src/components/Widgets/ReadMeCard/ReadMeCard.tsx

```diff
@@ -123,16 +123,16 @@ const ReadMeCard = (props: Props) => {
       >
         <MarkdownContent
           content={b64DecodeUnicode(value.content).replace(
-            /\[([^\[\]]*)\]\((?!https?:\/\/)(.*?)(\.png|\.jpg|\.jpeg|\.gif|\.webp)\)/gim,
+            /\[([^\[\]]*)\]\((?!https?:\/\/)(.*?)(\.png|\.jpg|\.jpeg|\.gif|\.webp)(.*)\)/gim,
```

Comment:
this could use a test tbh, it's getting very complex and will be brittle to modify

Reply:
➕